frontend/appflowy_flutter/lib/user/presentation/folder/folder_widget.dart

```diff
@@ -240,12 +221,70 @@ class CreateFolderWidgetState extends State<CreateFolderWidget> {
 
 Widget _buildTextButton(
     BuildContext context, String title, VoidCallback onPressed) {
-  return SizedBox(
-    width: 70,
-    height: 36,
-    child: RoundedTextButton(
-      title: title,
-      onPressed: onPressed,
-    ),
+  return FlowyTextButton(
+    title,
+    onPressed: onPressed,
+    fillColor: Theme.of(context).colorScheme.primary,
+    fontColor: Theme.of(context).colorScheme.onPrimary,
+    hoverColor: Theme.of(context).colorScheme.primaryContainer,
   );
 }
+
+class _FolderCard extends StatelessWidget {
```

Comment:
API looks similar to `ListTile`; why not use that instead?

Reply:
Because of how ListTile layout works, it's basically why, I assume, the person who created this screen in the first place, used SizedBoxes for buttons.

I've tried to create something in figma to illustrate, the first two are ListTiles, the final two are the widget specific to this usecase:
![image](https://user-images.githubusercontent.com/42929161/229051722-b89c9e3b-5986-4104-9d22-1bc99eddf5be.png)

Reply:
Fomr the documentation of ListTile:
`"Note that [leading](https://api.flutter.dev/flutter/material/ListTile/leading.html) and [trailing](https://api.flutter.dev/flutter/material/ListTile/trailing.html) widgets can expand as far as they wish horizontally, so ensure that they are properly constrained."`
https://api.flutter.dev/flutter/material/ListTile-class.html

Reply:
This is excellent, thank you 👍